shared/output/cloud/language_v1/lib/google/cloud/language/v1/language_service/client.rb

```diff
@@ -1284,13 +1299,19 @@ def annotate_text request, options = nil
             # @!attribute [rw] quota_project
             #   A separate project against which to charge quota.
             #   @return [::String]
+            # @!attribute [rw] universe_domain
+            #   The universe domain within which to make requests. This determines the
+            #   default endpoint URL. The default value of nil uses the environment
+            #   universe (usually GDU).
```

Comment:
's/GDU/the default "googleapis.com" universe/ or something to that effect.

Reply:
done